### DISCUSSION TITLE
Remote connection error

### DISCUSSION THREAD

**Discussion by mrtexperiments** [visose/Robots#269] [Q&A] [answered]
Good morning,

I am starting this discussion because I am currently working (experimenting at the moment) with a ABB robot - IRB6640 with `RobotWare 5.16. Everything works perfectly fine with Robots plugin but when I try to do a remote connection with the component "Remote" I get the following message:

1. Solution exception:Could not load type 'ABB.Robotics.Controllers.ConnectionType' from assembly 'ABB.Robotics.Controllers.PC, Version=6.7.7936.1011, Culture=neutral, PublicKeyToken=null'.

I just wanted to know if you had any idea on how to solve this?

Many thanks!

**Comment by visose** [comment, marked as answer]
The Robots plugin includes the file `ABB.Robotics.Controllers.PC.dll`, which should be the correct version. The error could happen if Rhino is loading a different version of this dll file. Check if you have other versions of Robots installed (you might have installed in manually) or you have another robot plugin that is including a different version of this file.

**Reply by mrtexperiments** [reply]
Hi Vicente, thank you for that. I'll try it as soon as possible.
